LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2020 Richard Ng
```

Comment:
Should the name be the Org name? "SE701ProjectGroup4"

Reply:
Yeah i was wondering about that since we don't have a project name

Reply:
I think Team Hong decided to have the organization name "Divergence Studios".

Reply:
Edited name to SE701ProjectGroup4